build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile 'com.itextpdf:itextpdf:5.5.9'
```

Comment:
implementation leaves dependencies as libs, but the idea was to pack everything into the "fat jar" to simplify execution from node

Reply:
I'm like 99.9999 %sure this can be done :smile: `compile` is deprecated and should not be used.
Did you try 
```
jar {
    manifest {
        attributes(
                'Main-Class': 'com.package.YourClass'
        )
    }
    from {
        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
    }
 }
```
?